Gems/Maestro/Code/Tests/Tracks/BoolTrackTest.cpp

```diff
@@ -6,117 +6,127 @@
+    TEST_F(CBoolTrackTest, GetValue_OneKey_AfterKey_ExpectNotDefault)
+    {
+        bool result = false;
+        m_singleKeyBoolTrack.GetValue(KEY_TIME, result);
 
-    //default value of m_singleKeyBoolTrack is true
-    EXPECT_TRUE(result) << "The track is not at default value, even though no keys have been hit yet";
-}
+        // default value of m_singleKeyBoolTrack is true
+        EXPECT_FALSE(result) << "Hitting a Key did not change the default value";
+    }
 
-TEST_F(CBoolTrackTest, GetValue_OneKey_AfterKey_ExpectNotDefault) 
-{
-    bool result = false;
-    m_singleKeyBoolTrack.GetValue(KEY_TIME, result);
+    TEST_F(CBoolTrackTest, GetValue_EvenKeys_ExpectDefault)
+    {
+        bool result = false;
 
-    //default value of m_singleKeyBoolTrack is true
-    EXPECT_FALSE(result) << "Hitting a Key did not change the default value";
-}
+        m_complexBoolTrack.GetValue(m_key1.time, result);
+        EXPECT_TRUE(result);
 
-TEST_F(CBoolTrackTest, GetValue_EvenKeys_ExpectDefault) 
-{
-    bool result = false;
+        m_complexBoolTrack.GetValue(m_key2.time, result);
+        // default value of m_complexBoolTrack is false
+        EXPECT_FALSE(result);
+    }
 
-    m_complexBoolTrack.GetValue(m_key1.time, result);
-    EXPECT_TRUE(result);
+    TEST_F(CBoolTrackTest, GetValue_OddKeys_ExpectNotDefault)
+    {
+        bool result = false;
+        m_complexBoolTrack.GetValue(m_key3.time, result);
+        // default value of m_complexBoolTrack is false
+        EXPECT_TRUE(result);
+    }
 
-    m_complexBoolTrack.GetValue(m_key2.time, result);
-    //default value of m_complexBoolTrack is false
-    EXPECT_FALSE(result);
-}
+    TEST_F(CBoolTrackTest, SetValue_SetDefault_ExpectChange)
+    {
+        m_emptyBoolTrack.SetValue(0.0f, /*value*/ false, /*set default?*/ true);
 
-TEST_F(CBoolTrackTest, GetValue_OddKeys_ExpectNotDefault) 
-{
-    bool result = false;
-    m_complexBoolTrack.GetValue(m_key3.time, result);
-    //default value of m_complexBoolTrack is false
-    EXPECT_TRUE(result);
-}
+        bool result = true;
+        m_emptyBoolTrack.GetValue(0.0f, result);
+        EXPECT_FALSE(result);
+    }
 
-TEST_F(CBoolTrackTest, SetValue_SetDefault_ExpectChange)
-{
-    m_emptyBoolTrack.SetValue(0.0f, /*value*/ false, /*set default?*/ true);
+    TEST_F(CBoolTrackTest, SetValue_DoNotSetDefault_NoChange)
+    {
+        m_singleKeyBoolTrack.SetValue(0.0f, /*value*/ false, /*set default?*/ false);
 
-    bool result = true;
-    m_emptyBoolTrack.GetValue(0.0f, result);
-    EXPECT_FALSE(result);
-}
+        bool result = false;
+        m_singleKeyBoolTrack.GetValue(0.0f, result);
+        EXPECT_TRUE(result);
+    }
 
-TEST_F(CBoolTrackTest, SetValue_DoNotSetDefault_NoChange) 
-{
-    m_singleKeyBoolTrack.SetValue(0.0f, /*value*/ false, /*set default?*/ false);
+} // namespace Maestro
 
-    bool result = false;
-    m_singleKeyBoolTrack.GetValue(0.0f, result);
-    EXPECT_TRUE(result);
-}
+#endif
```

Comment:
Its not showing up in github, so I can't tell if this is the case or not, but we do have an extra newline at the end of each of these files that have been touched at the end, right?  Files should end in a newline.  I know its silly but it causes problems otherwise.